packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2417,13 +2417,20 @@ is required to delete the user.
 | id        | `id` of user                                                                          | URL        | String   | TRUE     |         |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    | FALSE   |
 
+> User deletion can also be performed asynchronously. To invoke asynchronous user deletion, pass an HTTP header
```

Comment:
Change:

> to > **Note:**

User deletion can also be > You can also perform user deletion asynchronously.

Reply:
Updated.